hhs_oauth_server/request_logging.py

```diff
@@ -20,96 +23,336 @@
+        - req_grant_type = Request grant type.
+        - req_header_accept_encoding = Req header info.
+        - req_header_content_length = Req header info.
+        - req_header_content_type = Req header info.
+        - req_header_host = Req header info.
+        - req_header_referrer = Req header info.
+        - req_header_user_agent = Req header info.
+        - req_post_grant_type = Post grant type.
+        - req_qparam_beneficiary = Info from HTTP Query Params.
+        - req_qparam_beneficiary = Info from HTTP Query Params.
+        - req_qparam_client_id = Info from HTTP Query Params.
+        - req_qparam__count = Info from HTTP Query Params.
+        - req_qparam_count = Info from HTTP Query Params.
+        - req_qparam_format = Info from HTTP Query Params.
+        - req_qparam__id = Info from HTTP Query Params.
+        - req_qparam_lastupdated = Info from HTTP Query Params.
+        - req_qparam_patient = Info from HTTP Query Params.
+        - req_qparam_response_type = Info from HTTP Query Params.
+        - req_qparam_startindex = Info from HTTP Query Params.
+        - req_qparam_type = Info from HTTP Query Params.
+        - req_redirect_uri = Request redirect URI.
+        - req_refresh_token_hash = Request refresh token hash.
+        - req_response_type = Response type.
+        - req_scope = Request scope
+        - req_share_demographic_scopes = Request bene share demo scopes.
+        - request_method = http request method type (GET/POST/etc)
+        - request_post_grant_type = Post grant type.
+        - request_scheme = The scheme of the request (http/https/etc)
+        - request_uuid = The UUID identifying the request.
+        - req_user_id = Request user id.
+        - req_user_username = Request username.
+        - resp_access_token_hash = Response AC info.
+        - resp_access_token_scopes = Response AC info.
+        - resp_app_id = Response AC info.
+        - resp_app_name = Response AC info.
+        - resp_app_require_demographic_scopes = Response AC info.
+        - resp_dev_id = Response AC info.
+        - resp_dev_name = Response AC info.
+        - resp_expires_in = Response AC info.
+        - resp_fhir_id = Response AC info.
+        - response_code = The response status code.
+        - resp_refresh_token_hash = Response refresh token info.
+        - resp_scope = Response refresh token info.
+        - resp_token_type = Response refresh token info.
+        - resp_user_id = Response refresh token info.
+        - resp_user_username = Response refresh token info.
+        - size = Size in bytes of the response.content
+        - start_time = Unix Epoch format time of the request processed.
+        - type = Label for type of log. Set to "request_response_middleware".
+        - user_id = Login user (or None) or OAuth2 API id.
+        - user = Login user (or None) or OAuth2 API username.
+        - user_username = Login user (or None) or OAuth2 API username.
```

Comment:
It would be good to document which fields are legacy fields that retained their name for compatibility, to delineate from `request_` vs `req_`, etc.

Reply:
@njdister I updated the comments with a "BB2-342" tag next to the newly added field names.
Also fixed a bug I introduced while renaming "request_post_grant_type" --> "req_post_grant_type".